docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,388 @@
 ---
-title: How to Contribute to Teleport's Documentation
-description: Follow this guide to get started contributing changes to Teleport's documentation
+title: How to Contribute to the Teleport Documentation
+description: Follow this guide to learn how to contribute changes to Teleport documentation.
 ---
 
-This guide describes the general workflow for making documentation changes.
-It assumes you already have Node >=14 and `yarn` installed.
+We encourage everyone with an interest in improving Teleport documentation 
+to contribute content, submit issues, and suggest changes.
+This guide describes the process for making changes to the documentation whether 
+you are an internal or external contributor.
+
+## Prerequisites
+
+Before you begin, verify that your environment meets the following prerequisites:
+
+- You have a code editor, a GitHub account, and experience using command-line 
+  programs, including `git` commands and command line options.
+- You have Node.js, version 18.x or newer, installed. 
+   Run `node --version` to check whether Node is installed.
+   If you need to install or update Node, see 
+   [Installing Node.js](https://nodejs.org/en/download/package-manager) 
+   for instructions to download and install Node.js using a package manager.
+- You have the `yarn` package manager installed.
+   Run `yarn --version` to check whether `yarn` is installed. 
+   If you need to install `yarn`, run the command `brew install yarn` or 
+   `npm install --global yarn` to add it to your local system.
 
-## Step 1/4 Set up your local environment
+## Step 1/5. Set up your local environment
 
-Clone the `gravitational/docs` repository via `git clone`:
+To set up a local environment for contributing to Teleport documentation:
 
-```code
-$ git clone https://github.com/gravitational/docs
-```
+1. Open a terminal shell on your computer.
+
+1. Clone the `gravitational/docs` repository by running the following command:
 
-Different versions of the documentation are organized into the `content`
-directory of `gravitational/docs`. Each subdirectory of `content` includes a git
-submodule for a different branch of `gravitational/teleport`.
+   ```code
+   $ git clone https://github.com/gravitational/docs --recurse-submodules
+   ```
 
-Navigate to the root of your local clone of the `gravitational/docs` repository
-and run the following command to populate the `content` directory:
+   This command clones the documentation submodules that correspond to the
+   different major release versions of the Teleport code. For example, there
+   are documentation submodules for the 11.x, 12.x, 13.x, and 14.x versions
```

Comment:
Should we say "at the time of this writing" since the set of submodules will change several times per year?